models/User.php

```diff
@@ -1,4 +1,6 @@
 <?php 
+// Permet d'avoir accès à la base de données
+require_once('Model.php');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5347246)